Application/EdFi.Ods.Api/Security/Authorization/Repositories/EntityAuthorizer.cs

```diff
@@ -238,7 +239,7 @@ bool IsCreateUpdateOrDelete(string requestAction)
                                 {
                                     FilterDefinition = x.FilterDefinition,
                                     FilterContext = x.FilterContext,
-                                    Result = x.FilterDefinition.AuthorizeInstance(authorizationContext1, x.FilterContext)
+                                    Result = x.FilterDefinition.AuthorizeInstance(authorizationContext1, x.FilterContext, s.AuthorizationStrategyName)
```

Comment:
Added this string parameter for the `AuthorizationStrategyName`; it gets used during problem details construction and is especially handy in relationship-based strategies.